packages/js-dpp/schema/dataContract/dataContractMeta.json

```diff
@@ -401,6 +401,15 @@
               },
               "type": {
                 "const": "object"
+              },
+              "keySecurityLevelRequirement": {
+                "type": "integer",
+                "enum": [
```

Comment:
Let's use string enums to make more friendly to users?

Reply:
I'd argue that having it as a number better - it's smaller, and works better for some languages enum structure

Reply:
Or, alternatively, we can do somewhat the same thing as we do with binary fields, and have it as a string enum in the schema, but serialize it as a uint, but that can be even more confusing

Reply:
I've looked up how modern versions of Java and C# serialize enums, and actually, it looks like they can handle that well nowadays. So my first argument doesn't apply I guess. uint is still smaller though, and we still need to sort on this value

Reply:
I guess this small size is not so important for Data Contract comparing with Document. I would vote for better UX in this case.

Reply:
Actually they will see numbers that's why I suggest to use labels

Reply:
Agreed to add a description at least.